modules/key-card/src/generateQrData.ts

```diff
@@ -149,6 +151,9 @@ export function generateQrData({
   userKeychain,
   userMasterKey,
 }: GenerateQrDataParams): QrData {
+  assert(userKeychain, 'userKeychain is required');
+  assert(backupKeychain, 'backupKeychain is required');
+  assert(coin, 'coin is required');
```

Comment:
exactly my point as above -- if there are fields expected by one flow and not the other, we should let typescript/static checking do the majority of the work for us